### DISCUSSION TITLE
How to create Lets Encrypt  SSL for Azure SQL Server on VM

### DISCUSSION THREAD

**Discussion by Ruchi-Bod** [rcl-ssl/docs#31] [Q&A]
Hi,
I need assistance in generating Lets Encrypt SSL certificate for SQL server on Azure VM.

**Comment by rcladmin** [comment]
You can create a certificate in the RCL portal. Then you can try either option:

1. Install the certificate directly on the SQL Server following : 

[SQL Server TLS/SSL](https://docs.microsoft.com/en-us/sql/database-engine/configure-windows/enable-encrypted-connections-to-the-database-engine?view=sql-server-ver15)

2. Install the certificate on an Application Gateway and connect the SQL Server VM as a backed server to the gateway following:  

[Application Gateway TLS/SSL](https://docs.microsoft.com/en-us/azure/application-gateway/create-ssl-portal)

I have not personally tested either of these solutions for an SQL Server. You can try it out and tell me if it works.
